common/src/main/java/com/wynntils/core/text/StyledText.java

```diff
@@ -137,6 +137,14 @@ public String getString(PartStyle.StyleType type) {
         return builder.toString();
     }
 
+    public String getString() {
```

Comment:
As a follow up, can you convert all `getString(PartStyle.StyleType.DEFAULT)` to use this?